lib/harvester/base.rb

```diff
@@ -2,17 +2,17 @@ module Harvester
   # An iota of abstraction for things a harvester must do.
   # The starting point is always one or more Authors.
   class Base
-    # @return [void]
+    # @return [nil]
     def harvest_all
       Author.where(active_in_cap: true, cap_import_enabled: true)
             .find_in_batches(batch_size: batch_size)
             .each { |batch| harvest(batch) }
     end
 
     # @param [Enumerable<Author>] _authors
-    # @return [void]
```

Comment:
`void` is the correct return type here. 
https://github.com/lsegal/yard/blob/master/docs/GettingStarted.md#declaring-types

The value returned is not guaranteed or meaningful, and specifically is not guaranteed to be nil.

Reply:
OK, thanks for that explanation and pointer to the doc about using `void`.  Seems weird that they recommend something commonly used in Java, for which the ruby equivalent is `nil`.  Seems to me like giving no type details at all would be better, in the case where it has no meaning or significance at all.  Anyhow, I can change this to keep the `void`.